src/main/java/org/reaktivity/specification/nukleus/oauth/internal/OAuthJwtKeys.java

```diff
@@ -82,70 +80,68 @@
         es256.put("y", "x_FEzRu9m36HLN_tue659LNpXW6pCyStikYjKIWI5a0");
         es256.put("d", "jpsQnnGQmL-YBIffH1136cspYG6-0iY7X1fCE9-E9LI");
 
-        RFC7515_ES256 = initECKeyPair(es256);
+        KeyPair rsKeyPair = null;
+        KeyPair esKeyPair = null;
+        try
+        {
+            rsKeyPair = initRSAKeyPair(rsa256);
+            esKeyPair = initECKeyPair(es256);
+        }
+        catch(Exception e)
+        {
+            // Utility
+        }
+        RFC7515_RS256 = rsKeyPair;
+        RFC7515_ES256 = esKeyPair;
     }
 
     private static KeyPair initRSAKeyPair(
-        Map<String, String> params)
+        Map<String, String> params) throws Exception
```

Comment:
Make sure the `throws` exception type is as specific as possible, not just generic `Exception`.
Please apply this feedback to all such `throws` in the method signatures called from this method too, same for `initECKeyPair`.